database/migrations/2025_08_26_000000_create_matchmaking_stats_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('matchmaking_stats', function (Blueprint $table) {
+            $table->unsignedInteger('user_id')->nullable(false);
+            $table->unsignedInteger('first_placements')->default(0);
```

Comment:
as discussed in discord, should have a `score` column for basic score tallying.

i'd also add a standard updated at column in line with other laravel tables for bookkeeping.